services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: str, movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
```

Comment:
You can set the id number without getting an instance (when you do, you load DB again)
Like: movie_id = movie_id